src/CMakeLists.txt

```diff
@@ -47,6 +47,13 @@ endif()
 
 include("list_of_files.cmake")
 
+
```

Comment:
Can we move this to the top level `CMakeLists.txt` so that any other place `ld` is being used, it will take effect.